### DISCUSSION TITLE
应用状态一直在重启中

### DISCUSSION THREAD

**Discussion by lizheming** [labring/laf#171] [Feedbacks]
应用状态一直在重启中，appId `ublozg`。
![image](https://user-images.githubusercontent.com/424491/178136463-16993588-38bc-462a-b865-ea57d85d8038.png)

**Comment by maslow** [comment]
我来排查一下

**Comment by maslow** [comment]
<img width="1353" alt="image" src="https://user-images.githubusercontent.com/7122237/178207320-1a13051d-6a6e-45ff-b474-a7e08bba3307.png">

@waline/vercel@latest 这个包依赖的成分在这个 node 环境下安装不了，有些问题，我需要进一步排查一下，是不是应用环境中 缺少 什么东西

**Reply by lizheming** [reply]
应该是 sqlite3 的依赖，需要 node-gyp 编译，看了下报错信息，应该是没有 Python 无对应的编译环境

**Reply by maslow** [reply]
嗯，我来重新做个 app service 镜像

**Reply by lizheming** [reply]
@maslow 请问这个镜像做好了么？

**Reply by maslow** [reply]
最新版的 app-service:0.8.7 已经可以正常安装 @waline/vercel@latest 这个依赖。
